### PR TITLE
Integration done

### PR DESCRIPTION
**Pull request by VireshReddy123**

### CODE REVIEW

**Review by AnuragRana2109 — APPROVED** [review]

File: src/main/java/org/automation/pageObjects/DirectorPage.java [hunk 1]
```diff
@@ -5,111 +5,157 @@
...
-    public By directorsLastName=By.xpath("//input[@placeholder='Last Name']");
-    public By directorsMobileNumber=By.xpath("//input[@placeholder='(999) 999-9999']");
-    public By directorsEmail=By.xpath("//input[@placeholder='Email']");
-    public By assignLocation=By.xpath("//select[@id='testingLocation']");
-    public By directorsLocationName=By.xpath("//option[text()='Plano']");
-    public By directorsUserName=By.xpath("//input[@placeholder='Username']");
-    public By password_Field=By.xpath("//input[@placeholder='Create Password']");
-    public By confirm_PasswordField=By.xpath("//input[@placeholder='Confirm Password']");
-    public By createDirectorsButton=By.xpath("//button[text()='Create Director']");
+    WebdriverWaits wait = new WebdriverWaits();
```
Comment (saksham-sqelabs) [inline src/main/java/org/automation/pageObjects/DirectorPage.java]: In between the xpath, why we are creating the instance of class?

**Comment by saksham-sqelabs** [comment]
Please correct the name of PR, Integration done for what ?

**Comment by saksham-sqelabs** [comment]
Git branch that you are creating from local should be informative, please don't use "VireshNew" , "changes of maik" etc.

File: src/main/java/org/automation/pageObjects/DirectorPage.java (hunk 1 above)
Comment (VireshReddy123) [reply src/main/java/org/automation/pageObjects/DirectorPage.java]: I have verified and that instance of class is not exist in current codes

**Comment by VireshReddy123** [comment]
> Please correct the name of PR, Integration done for what ?

Integration donefor superAdmin, admin,and diagnostician

File: src/main/java/org/automation/pageObjects/DiagnosticianPage.java [hunk 2]
```diff
@@ -472,7 +474,7 @@ public void click_Slot() throws InterruptedException {
         for (WebElement slot : slots) {
             Thread.sleep(2000);
             moveToEleByWE(slot);
-            if (getDriver().findElements(By.xpath("//div[@class='ng-star-inserted']")).size() > 4) {
```
Comment (SKaur2400) [inline src/main/java/org/automation/pageObjects/DiagnosticianPage.java:475]: fetch from config

**Pull request merged**